src/installer.ts

```diff
@@ -64,11 +64,16 @@ async function cacheBazelisk(
   const downloadUrl: string = `${downloadPrefix}/${info.tag_name}/${osFileName}`;
   core.info(`Acquiring ${info.tag_name} from ${downloadUrl}`);
   const auth = `token ${token}`;
```

Comment:
If I understand correctly the difference between GitHub and Gitea, then the `${token}` parameter here will be empty.
As such, instead of the code changes below you can probably just make the following change here:
```suggestion
  const auth = token ? `token ${token}` : undefined;
```

Please test this, as I'm not sure what comes in `${token}` in Gitea.

Reply:
`${token}` not empty or null. `${token}`, and it was not `github token`, currently(`Oct 27 2023`) cannot be customized.